boards/stm32f746g-disco/doc.txt

```diff
@@ -0,0 +1,27 @@
+/**
+ * @defgroup    boards_stm32f746g-disco STM32F746G-DISCO
+ * @ingroup     boards
+ * @brief       Support for the STM32F746G-DISCO board.
+
```

Comment:
Some information about what this device is (no need for a full enumeration, but maybe a vendor link) would be nice, and a description of which components are and are not supported.

Reply:
I agree. Is it ok for you if I do it in a follow-up PR ?

Reply:
Sure.

Reply:
Thanks, will try to find some to add that asap.